public/api/v1/incompatible-mods.json

```diff
@@ -122,5 +122,22 @@
     "tracking": "UNKNOWN",
     "note": "Compostable items cannot be composted on Quilt",
     "icon": "https://cdn.modrinth.com/data/HJetCzWo/baf9ea57b7784e743aa33441105723205d743351.png"
-  }	 	  
+  },
+  {
+    "ids": ["requiem"],
+    "name": "Requiem",
+    "type": "WORKAROUND",
+    "status": "WONT_FIX",
```

Comment:
The issue you've linked in the `tracking` field doesn't have a response from the mod authors saying they won't fix it. Should this be a `NO_ANSWER`?
```suggestion
    "status": "NO_ANSWER",
```

Reply:
Maybe? Up themselves has provided this solution at several points in time, I might try to directly reach out to Pyrofab/Up to reach a conclusive answer on that.

Reply:
that'd be on @Pyrofab to give a definitive answer..

Reply:
Pyrofab provided a direct reply in the Ladysnake Discord. `WONT_FIX` is the tag that should be used.

![ResizedImage_2024-11-22_04-07-12_2433.png](https://github.com/user-attachments/assets/04ef456c-3bfd-4004-bb76-7c48997489e2)